README.md

```diff
@@ -2,26 +2,30 @@
 
 ## Prerequisites
 
+### Install the Go compiler
+
+You'll need to have a recent version of the Go language compiler installed locally: https://go.dev/
+
+### Install Protobuf and gRPC
+
+```sh
+brew install grpc protobuf
```

Comment:
Should we combine this `brew install` with the `brew install temporal` below?

Reply:
Yes, this is a good idea.